README.md

```diff
@@ -83,6 +83,38 @@ You can check out [the Next.js GitHub repository](https://github.com/vercel/next
 
 </details>
 
+## Collaboration: Want to collaborate with this project ?
+
+<details>
+<summary> 
+STEPS
+</summary>
+
+1. Forked the project repository from the owners repository.
```

Comment:
```suggestion
1. Fork the project repository from this repository.
```

Reply:
I have corrected  the suggested changes.

Reply:
It still shows the original text @Amutha37 , do you want to push the changes?

Reply:
Hi @Amutha37 ,

I think this conversation has gone on unresolved for too long. I'll resolve this for now, and address outstanding work in a separate PR.